book.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'rentals'
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    # rental = Rental.new(date, self, person)
+    @rentals << rental
+    # person.rentals << rental
+    rental.person.rentals << rental
+  end
+
+  attr_reader :rentals
```

Comment:
- [ ] Kindly remove the getter for `:rentals` as what we should have is both getter and setter as represented by `attr_accessor`.

Reply:
Fixed! 🔨

Reply:
This has been done well